src/components/confirm/confirm.pw.tsx

```diff
@@ -603,6 +637,42 @@ test.describe("should check accessibility for Confirm", () => {
 
     await checkAccessibility(page);
   });
+
+  ([["before"], ["after"]] as [
```

Comment:
Suggestion: As there's only one element in the inner arrays here you can flatten it. Maybe it's worth storing it in an variable such as `const iconPositions:  [ConfirmProps["cancelButtonIconPosition"]][] = ["before", "after"];`

```
iconPositions.forEach((position) => {
    test(`should check accessibility when icon on Cancel button is ${position} text`, async ({
      mount,
      page,
    }) => {
      await mount(
        <ConfirmComponent
          cancelButtonIconType="bin"
          cancelButtonIconPosition={position}
        />
      );

      await checkAccessibility(page);
    });
  });